hw/top_earlgrey/formal/top_earlgrey_fpv_cfgs.hjson

```diff
@@ -333,6 +333,14 @@
                rel_path: "hw/ip/kmac/sha3pad/{sub_flow}/{tool}"
                cov: true
              }
+             {
+               name: prim_max_tree_fpv
+               dut: prim_max_tree_tb
+               fusesoc_core: lowrisc:fpv:prim_max_tree_fpv
```

Comment:
If we do not have a FPV testbench, you can do:
```
  name: prim_max_tree
  dut: prim_max_tree
  fusesoc_core:  lowrisc:prim:max_tree
```
I think that should run the prim_max_tree module with all the assertions under its RTL file.

Reply:
Let me try this.

Reply:
Taken care of here: https://github.com/lowRISC/opentitan/pull/10555